pymisp/mispevent.py

```diff
@@ -1385,6 +1398,7 @@ def add_object(self, obj: Union[MISPObject, dict, None]=None, **kwargs) -> MISPO
             misp_obj.from_dict(**kwargs)
         else:
             raise InvalidMISPObject("An object to add to an existing Event needs to be either a MISPObject, or a plain python dictionary")
+        misp_obj.standalone = False
```

Comment:
not using the setter function?

Reply:
`standalone` is decorated by `property`, so setting it calls the function decorated by `standalone.setter`